pkgs/by-name/bb/bbox-services/package.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  stdenv,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  protobuf,
+  darwin,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "bbox-services";
+  version = "0.6.1";
+
+  src = fetchFromGitHub {
+    owner = "bbox-services";
+    repo = "bbox";
+    rev = "v${version}";
+    hash = "sha256-NI6EL9UPqv1OtvEySGpk1jfHOykOvkTELkH+qR3dL3w=";
+  };
+
+  cargoHash = "sha256-ofby7fHgybRE4inx0n28glwqHcddKTo+hKN/JkmR22I=";
```

```suggestion
  useFetchCargoVendor = true;
  cargoHash = "sha256-azZ67e+6JwaGuxBwvdq1l9GtenFuMqIHOOpqnoRv+II=";
```
The nix PR is 100% unrelated. I think it is actually a bug in the default fetcher, so we can just use the new one.